packages/dashboard/src/features/compute/components/ServiceCard.tsx

```
@@ -91,16 +91,26 @@ export function ServiceCard({ service, onClick, onStop, onStart, onDelete }: Ser
       {reachableUrl && (
-        <a
-          href={reachableUrl}
-          target="_blank"
-          rel="noopener noreferrer"
-          onClick={(e) => e.stopPropagation()}
-          className="inline-flex items-center gap-1 text-xs text-primary hover:underline mb-3"
-        >
-          <ExternalLink className="h-3 w-3" />
-          {reachableUrl}
-        </a>
+        reachableUrl.href ? (
```

ESLint

[error] Insert `(`

(prettier/prettier)